src/plugin.ts

```diff
@@ -574,14 +632,14 @@ function insertTransformedTemplate<EnvSpecificOptions>(
   let transformed = print(ast, { entityEncoding: 'raw' });
   if (target.isCallExpression()) {
     (target.get('arguments.0') as NodePath<t.Node>).replaceWith(t.stringLiteral(transformed));
-    if (!scopeLocals.isEmpty()) {
```

Comment:
scope locals could have changed from something, to empty, we need to remove existing scope